playbooks/common/openshift-docker/config.yml

```diff
@@ -0,0 +1,85 @@
+- name: Set facts necessary to configure docker
+  hosts:
+    - masters
+    - nodes
```

Comment:
Since there is already a docker role, then an openshift-docker role that encapsulates that leverages the docker role may be the way to go. Anything generic (like the {add,block,insecure}-registry settings) can go in the docker role, anything more openshift specific in openshift-docker.